compiler/tests-common/tests/org/jetbrains/kotlin/TestHelperGenerator.kt

```diff
@@ -188,3 +190,11 @@ fun createTextForHelpers(isReleaseCoroutines: Boolean, checkStateMachine: Boolea
             |${if (checkTailCallOptimization) checkTailCallOptimizationString else ""}
         """.trimMargin()
 }
+
+// Add the directive `// WITH_HELPERS` to use these helpers in codegen tests (see CodegenTestCase.java).
+fun createTextForCodegenTestHelpers(backend: TargetBackend) =
+    """
+        |package helpers
+        |
+        |fun getTargetBackend() = "${backend.name}"
```

Comment:
Maybe `fun isIR(): Boolean = "${backend.isIR}"` instead? Seems less error-prone than to compare strings, where you can have a typo...

Reply:
Ok, changed. I suppose if we need to check for specific backends in the future, we can add the method back.